charts/kong/values.yaml

```
@@ -368,7 +368,14 @@ ingressController:
     periodSeconds: 10
     successThreshold: 1
     failureThreshold: 3
+  # Remove "{}" and uncomment the block below to set controller resources
   resources: {}
+  # limits:
+  #  cpu: 100m
+  #  memory: 256Mi
+  # requests:
+  #  cpu: 50m
+  #  memory: 128Mi
```

```suggestion
  resources: {}
  # Example reasonable setting for "resources":
  # resources:
  #   limits:
  #     cpu: 100m
  #     memory: 256Mi
  #   requests:
  #     cpu: 50m
  #     memory: 128Mi
```

Two comments here:
1. One does not edit this `values.yaml` (embedded in the chart) directly (and we don't recommend copying it entirely, but creating a minimal derivative), so the notion of manipulating parts of this file seems slightly misleading. I recommend an alternative of giving a complete example of the "resources" block.
2. Indentation: let's be consistent with the rest of the file on "2 spaces per level"